doc-content/enterprise-only/springboot/openshift-springboot-proc.adoc

```diff
@@ -0,0 +1,105 @@
+ifdef::PAM[]
+* If you want to use {CENTRAL} Monitoring, you installed {CENTRAL} Monitoring using the operator. For instructions about installing {CENTRAL} Monitoring and other {PRODUCT} components using the operator, see {URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-operator[_{DEPLOYING_OPENSHIFT_OPERATOR}_].
++
+[IMPORTANT]
+====
+You must configure {CENTRAL} Monitoring to use the controller startup strategy. To enable the controller strategy on {CENTRAL} Monitoring, in the *Console* tab of the operator configuration, add a `KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED` environment variable and set it to `false`.
+====
+endif::PAM[]
+
+.Procedure
+
+. Outside the business application project directories, create an `ocp-image` directory with the following subdirectories:
++
+----
+ocp-image
+|--/root
+  |--/opt
+    |-- /spring-service
+----
++
+. Copy the single JAR file for your business application into the `root/opt/spring-service` subdirectory, for example:
++
+----
+cd ../business-application-service
+cp target/business-application-service-1.0-SNAPSHOT.jar ../ocp-image/root/opt/spring-service/
+----
++
+. In the `ocp-image` directory, create a `Dockerfile` file with the following content:
++
+----
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
+COPY root /
+EXPOSE 8090
+WORKDIR /opt/spring-service/
+CMD ["sh","-c", "java ${JAVA_OPTIONS} -Dorg.kie.server.mode=PRODUCTION --jar /opt/spring-service/FILENAME.jar"]
+----
++
+Replace `FILENAME.jar` with the name of the single JAR file for your business application.
++
+ifdef::PAM[]
+Alternatively, if you want to configure communication with {CENTRAL} Monitoring, use the following content for `Dockerfile`:
++
+----
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
+COPY root /
+EXPOSE 8090
+WORKDIR /opt/spring-service/
+CMD ["sh","-c", "java ${JAVA_OPTIONS} -Dorg.kie.server.controller=${KIESERVER_CONTROLLERS} -Dorg.kie.server.controller.user=${KIE_SERVER_CONTROLLER_USER} -Dorg.kie.server.controller.pwd=${KIE_SERVER_CONTROLLER_PWD} -Dorg.kie.server.mode=PRODUCTION -Dorg.kie.server.startup.strategy=LocalContainersStartupStrategy -Dkie.maven.settings.custom=/opt/m2/settings.xml --jar /opt/spring-service/FILENAME.jar"]
+----
++
+Replace `FILENAME.jar` with the name of the single JAR file for your business application.
++
+endif::PAM[]
+. To build the initial image and deploy it in your {OPENSHIFT} environment, complete the following steps:
+.. To build the image, run the following commands in the `ocp-image` directory:
++
+----
+oc new-build --binary --strategy=docker --name openshift-kie-springboot
+oc start-build openshift-kie-springboot --from-dir=. --follow
+----
++
+Optionally, replace `openshift-kie-springboot` with a custom application name in these commands and all subsequent commands.
++
+.. To deploy the image in the {OPENSHIFT} environment, run the following command:
++
+----
+oc new-app openshift-kie-springboot
+----
```

Comment:
I would also suggest to add there command for exposing route of this service. The port is needed, because it's used in Dockerfile. By default expose command is creating route on service for port 8080.
```suggestion
oc new-app openshift-kie-springboot
oc expose service/openshift-kie-springboot --port=8090
----
```